rfc9114.md

```diff
@@ -2026,12 +2027,12 @@ in certain scenarios.
 # IANA Considerations
 
 This document registers a new ALPN protocol ID ({{iana-alpn}}) and creates new
-registries that manage the assignment of codepoints in HTTP/3.
+registries that manage the assignment of code points in HTTP/3.
 
 ## Registration of HTTP/3 Identification String {#iana-alpn}
 
 This document creates a new registration for the identification of
-HTTP/3 in the "Application-Layer Protocol Negotiation (ALPN)
+HTTP/3 in the "TLS Application-Layer Protocol Negotiation (ALPN)
```

Comment:
and then it's not hyphenated here..

Reply:
I'm confused -- it _is_ hyphenated here.  Or do you mean they didn't remove the hyphen?  Regardless, the name of the registry is "TLS Application-Layer Protocol Negotiation (ALPN) Protocol IDs", so I agree that we should match.

Reply:
gah I can't type - Muphry's Law strikes. Making them both hyphenated is my suggestion.